app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,69 @@
+/* mixins, variables, etc. */
```

Comment:
yep, completely agree. It's a stub and can be destroyed once we have proper CSS architecture.